.golangci.yaml

```diff
@@ -11,6 +11,18 @@ linters:
     - unparam
   disable:
     - errcheck
+  settings:
+    gosec:
+      excludes:
+        - G115
+    staticcheck:
+      checks:
+        - all
+        - "-ST1019"
+        - "-ST1003"
+        - "-ST1012"
+        - "-ST1016"
+        - "-ST1001"
```

Comment:
Ignores for a lot of staticcheck issues due to 100+ occurrences most for name formats. I've tried to fix other most annoying issues like deprecated packages and functions.

Something might break thought. :)